src/tool_launcher/tool_launcher.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Targets .NET Framework 3.5 because .NET Framework 4 is not bundled in Windows 7. -->
+    <TargetFrameworks>net45;net35</TargetFrameworks>
```

Comment:
Sorry to come in after the review, but I don't get why we're multi-targeting. Why can't we just target net35 (minreq) and put a config that allows running on 45 when 35 is not installed? Feels like the code would be much simpler.

Reply:
I was under the impression net35 apps wouldn't automatically roll forward to net45 if net35 isn't present. How do you configure an app to do this?

Reply:
https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-configure-an-app-to-support-net-framework-4-or-4-5

> You can add multiple `<supportedRuntime>` elements, listed in order of preference, to specify support for multiple versions of the .NET Framework.

Reply:
Cool, didn't know that worked. => #8433